ui/conftest.py

```diff
@@ -25,15 +25,20 @@
 """
 
 
-def pytest_addoption(parser):
-    parser.addoption("--network", action="store", default="devnet", help="Which stand use")
-
-
 @pytest.fixture(scope="session")
 def network(pytestconfig: tp.Any) -> tp.Optional[str]:
     return EVM_NETWORKS.get(pytestconfig.getoption("--network"), EVM_NETWORKS["devnet"])
 
 
+@pytest.fixture(scope="session")
+def neonpass_url(pytestconfig: tp.Any) -> tp.Optional[str]:
```

Comment:
I think we could move to top level conftest.py from integration directory and don't repeat common init code

Reply:
Do you mean to move the `operator` fixture from integration directory to the root conftest file, and also include `neonpass_url` into it?

Reply:
@lyubenkov in integration we already have code to initialize environment, read envs.json and more. In UI directory we do the same. Common parts of code move to root, specific left in integration and ui

Reply:
Discussed and came to agreement, that we don't need to change anything here as it's already great!